test/src/runner/cucumber-integration/testCucumberTestsInParallel.js

```diff
@@ -6,6 +6,8 @@ const MockServer = require('../../../lib/mockserver.js');
 const {runTests} = common.require('index.js');
 
 describe('Cucumber integration - parallel running', function() {
+  this.timeout(10000);
```

Comment:
why we had added a timeout here?

Reply:
When running with coverage, every line of code gets instrumented with coverage tracking code. This makes the execution time double. Which causes the test to cross the 2000ms default timeout for mocha. Hence I increased the default timeout.